docker-app/qfieldcloud/core/serializers.py

```diff
@@ -535,3 +546,44 @@ class LatestPackageSerializer(serializers.Serializer):
     package_id = serializers.UUIDField()
     packaged_at = serializers.DateTimeField()
     data_last_updated_at = serializers.DateTimeField()
+
+
+class TeamSerializer(serializers.ModelSerializer):
+    organization = serializers.StringRelatedField(source="team_organization")
+
+    class Meta:
+        model = Team
+        fields = ("username", "organization")
+
+
+class TeamMemberSerializer(serializers.ModelSerializer):
+    member = serializers.CharField()
+
+    def to_internal_value(self, data: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
```suggestion
    def to_internal_value(self, data: dict[str, Any]) -> dict[str, Any]:
```

Since python 3.9 you can use the collections directly, check https://peps.python.org/pep-0585/